analysis/tools/ccsh/src/main/kotlin/de/maibornwolff/codecharta/tools/ccsh/Ccsh.kt

```diff
@@ -133,9 +135,39 @@ class Ccsh : Callable<Void?> {
                 return finalExitCode
             }
             // Improvement: Try to extract merge commands before so user does not have to configure merge args?
-            logger.info("Each parser was successfully executed and created a cc.json file. " +
-                        "You can merge all results by making sure they are in one folder and executing the merging tool.")
-            return 0
+            logger.info("Each parser was successfully executed and created a cc.json file.")
+            return askAndMergeResults(commandLine)
+        }
+
+        private fun askAndMergeResults(commandLine: CommandLine): Int {
+            val shouldMerge = KInquirer.promptConfirm(
+                    message = "Do you want to merge all generated files into one result now?",
+                    default = false)
+
+            return if (shouldMerge) {
+                val ccJsonFilePath = KInquirer.promptInput(
+                        message = "What is the folder path containing all cc.json files?",
+                        hint = "If you did not output all cc.json files into the same folder, " +
+                              "you need to manually move them there before trying to merge.")
+
+                val outputFilePath = "$ccJsonFilePath/mergedResult.cc.json"
+                // Default args with input path being the output path as well
+                val mergeArguments =
+                        listOf(
+                                ccJsonFilePath,
+                                "--output-file=$outputFilePath+",
+                                "--not-compressed=true",
+                                "--add-missing=false",
+                                "--recursive=true",
+                                "--leaf=false",
+                                "--ignore-case=false"
+                              )
+
+                val map = mapOf(InteractiveParserHelper.MergeFilterConstants.name to mergeArguments)
+                executeConfiguredParser(commandLine, map.entries.first())
```

Comment:
I changed it a bit, it now returns the result of the function call directly.
Is that fine with you?